File: EjemploPOO.cpp
class Contacto
{	//Atributos de contacto
	public char nombre[30];
	public long telefono;

	//Métodos Contacto
	public Contacto(char n[30], long t){
	strcpy(this.nombre,n);
	this.telefono = t;
}

	void Escribir(){
	printf("Nombre: %s telefono: %ld\n", this.nombre, this.telefono);
}

	void CambiarTelefono(long nt){
	this.telefono = nt;
	
}
}
main()
{
	Contacto c1;

	c1 = new Contacto("Pepe", 1111);
	c1.Escribir();
	c1.CambiarTelefono(22222);	

	
};

File: HTML/Javascript/Scripts/EjemploPOO.js
class Contacto
{
	// Creación del constructor
	constructor(n,t)
	{
		this.nombre = n;
		this.telefono = t;
	}

	CambiarNombre(nuevoV)
	{
		this.nombre = nuevoT;
	}

	Escribir()
	{
		var resultado;

		resultado = document.getElementById('Resultado');
		resultado.innerHTML = "Nombre: " + this.nombre + "<br>" + "Teléfono: " + this.telefono + "<br>";
	}

	CambiarTelefono(nuevoT)
	{
		this.telefono = nuevoT;
	}
//Objetos globales
	var c1;
	function CrearContacto() {
		c1 = new Contacto("Pepe", 1111);
		
		c1.Escribir();
	}

	function ModificarContacto()
	{
		var valorNombre,
			valorTelefono;

		valorNombre = document.getElementById('Nombre').value;
		valorTelefono = document.getElementById('Telefono').value;
		c1.CambiarNombre(valorNombre);
		c1.CambiarTelefono(valorTelefono);
		c1.Escribir();
	}
}